index.html

```diff
@@ -38,7 +48,8 @@ <h2 class="connect"><a href="#">LET'S CONNECT</h2></a>
     </ul>
         </div>
     </section>
-    <section class="projects">
+</div>
+    <section class="projects" id="#Portfolio">
```

Comment:
- [ ] Please I was not redirected to the appropriate section when I clicked on the `nav links`. Kindly address it. 👍 

```suggestion
</div>
    <section class="projects" id="Portfolio">
```